api/src/main/java/org/ehcache/expiry/Expirations.java

```diff
@@ -45,6 +45,19 @@
   }
 
   /**
+   * Get a time-to-live (TTL) {@link Expiry} instance for the given {@link java.time.Duration}.
```

Comment:
I think we should deprecate the old methods use our duration object. And in fact, would prefer that we convert to `java.time.Duration` instead of the other was around.

Reply:
+1